apps/condo/domains/banking/schema/BankTransaction.test.js

```diff
@@ -22,21 +22,32 @@ const {
 } = require('@condo/domains/banking/utils/testSchema')
 const { createTestOrganization, createTestOrganizationEmployeeRole, createTestOrganizationEmployee, createTestOrganizationLink } = require('@condo/domains/organization/utils/testSchema')
 const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
+const { makeClientWithServiceUser } = require('@condo/domains/user/utils/testSchema')
 
 const { BANK_INTEGRATION_IDS } = require('../constants')
-const { createTestBankCategory, createTestBankCostItem } = require('../utils/testSchema')
+const { createTestBankCategory, createTestBankCostItem, BankIntegrationAccessRight, createTestBankIntegrationOrganizationContext } = require('../utils/testSchema')
 
 let admin
 let support
+let serviceClient
 let anonymous
 let bankIntegration
+let SBBOLBankIntegration
 
 describe('BankTransaction', () => {
     beforeAll(async () => {
         admin = await makeLoggedInAdminClient()
         support = await makeClientWithSupportUser()
         anonymous = await makeClient()
         bankIntegration = await BankIntegration.getOne(admin, { id: BANK_INTEGRATION_IDS['1CClientBankExchange'] })
+        SBBOLBankIntegration = await BankIntegration.getOne(admin, { id: BANK_INTEGRATION_IDS.SBBOL })
+        serviceClient = await makeClientWithServiceUser()
+        await BankIntegrationAccessRight.create(admin, {
+            user: { connect: { id: serviceClient.user.id } },
+            integration: { connect: { id: SBBOLBankIntegration.id } },
+            dv: 1,
+            sender: { dv: 1, fingerprint: 'tests' },
```

Comment:
Why not to use `createTestBankIntegrationAccessRight` generated test util?
This way you don't need to specify `sender`